### PR TITLE
ui:fixed bug where license was showing in oss

### PR DESCRIPTION
**Pull request by jherschman**
Noticed bug where license tab was still showing for OSS users. Worked with @johncowen to update to check to ensure license tab only shows for enterprise users.

### CODE REVIEW

File: ui/packages/consul-ui/app/abilities/license.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+import { inject as service } from '@ember/service';
 
 export default class LicenseAbility extends BaseAbility {
   resource = 'operator';
   segmented = false;
+
+  @service('env') env;
+
+  get canRead() {
+    return this.env.var('CONSUL_NSPACES_ENABLED') && super.canRead;
```
Comment (evrowe) [inline ui/packages/consul-ui/app/abilities/license.js:11]: @jherschman For my clarification, the `CONSUL_NSPACES_ENABLED` env var is the one that we use to determine whether a customer is OSS or Ent?

Comment (jherschman) [reply ui/packages/consul-ui/app/abilities/license.js:11]: @evrowe yea that's what I have been enabling to to QA enterprise!  If you want / dont already have you can create a bookmark with this if you want an easy way to toggle it on: ` javascript:Scenario('CONSUL_NSPACES_ENABLE=1')`

**Review by evrowe — APPROVED** [review]
looks good!

**Review by jkirschner-hashicorp — APPROVED** [review]
Double checked @evrowe 's small tweak. LGTM.

**Pull request merged**